google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -687,7 +714,7 @@ com.google.datastore.v1.BeginTransactionResponse beginTransaction(
     com.google.cloud.datastore.telemetry.TraceUtil.Span span =
         otelTraceUtil.startSpan(
             com.google.cloud.datastore.telemetry.TraceUtil.SPAN_NAME_BEGIN_TRANSACTION,
-            otelTraceUtil.getCurrentContext());
+            otelTraceUtil.getCurrentSpanContext());
```

Comment:
Do we need to pass the same `otelTraceUtil.getCurrentSpanContext()` to L689?

Reply:
Sorry, I didn't understand the comment.